lib/Doctrine/ODM/MongoDB/Aggregation/Expr.php

```diff
@@ -333,6 +333,14 @@ public static function convertExpression($expression)
         return $expression;
     }
 
+    /**
+     * Returns a new expression object
+     */
+    public function createExpr(): self
```

Comment:
`createAggregationExpression` for clarity:
```suggestion
    /**
     * Returns a new expression object
     *
     * @return static
     */
    public function createAggregationExpression(): self
```